### DISCUSSION TITLE
Is it possible to scroll to the first message but without `ScrollToUnreadOptions`

### DISCUSSION THREAD

**Discussion by TJMusiitwa** [flyerhq/flutter_chat_ui#555] [Ideas & Requests]
We already have the amazing scroll to unread feature in the library which is amazing. 
However, for the implementation I am trying to get, I would like to open a text chain transcript to the first message without showing the **"Unread Messages"** widget i.e. Instead of `ScrollToUnreadOptions` there is like `scrollToFirstMessage:true`

Is there any way possible within the confines of v1 of this library to do so. 

`ScrollToUnreadOptions` is very, very close to what I would need but if there was a way to remove the unread messages widget and open/scroll to the very first message.

Thanks

**Comment by demchenkoalex** [comment]
opened an issue from this to not forget
